docs/telemetry.md

```diff
@@ -1,53 +1,386 @@
+
+You control whether to share this information. Participation in this program is completely voluntary. If don't want to share anonymous data, you can [disable telemetry](#disable-telemetry).
+
+## Why telemetry matters
+
+Benefits for Percona:
+
+| Advantages                  | Description                                                                                                                                                         |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| See how people use your software | Telemetry collects anonymous data on how users interact with our software. This tells developers which features are popular, which ones are confusing, and if anything is causing crashes. |
+| Identify issues early       | Telemetry can catch bugs or performance problems before they become widespread. |
+
+Benefits for users in the long run:
+
+| Advantages                | Description                                                                                                         |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------|
+| Faster bug fixes          | With telemetry data, developers can pinpoint issues affecting specific users and prioritize fixing them quickly.  |
+| Improved features           | Telemetry helps developers understand user needs and preferences. This allows them to focus on features that will be genuinely useful and improve your overall experience. |
+| Improved user experience  | By identifying and resolving issues early, telemetry helps create a more stable and reliable software experience for everyone. |
+
+## Telemetry components
+
+Percona collects information using the following components:
+
+* Telemetry script that sends the information about the software and the environment where it is installed. This information is collected only once during the installation.
+
+* The `percona_pg_telemetry` extension collects the necessary metrics directly from the database and stores them in a Metrics File.
+
+* The Metrics File stores the metrics and is a standalone file located on the database host's file system.
+
+* The Telemetry Agent is an independent process running on your database host's operating system and carries out the following tasks:
+
+    * Collects OS-level metrics
+
+    * Reads the Metrics File, adds the OS-level metrics
+
+    * Sends the full set of metrics to the Percona Platform
+
+    * Collects the list of installed Percona packages using the local package manager
+
+The telemetry also uses the Percona Platform with the following components:
```

Comment:
I am not sure on the status of a product called Percona Platform but it seems that this mention here refers to it while I am almost certain it does not